Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -123,13 +110,15 @@ CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedStrin
     }
 
     // Get the default locale for this user.
-    std::wstring localeName = _GetUserDefaultLocaleName();
+    // If the default locale is returned, find that locale name, otherwise use "en-us".
+    wchar_t localeNameBuf[LOCALE_NAME_MAX_LENGTH];
+    int defaultLocaleSuccess = GetUserDefaultLocaleName(localeNameBuf, LOCALE_NAME_MAX_LENGTH);
+    const wchar_t* localeName = defaultLocaleSuccess ? localeNameBuf : c_defaultUserLanguage;
 
```

Comment:
More importantly, is this really that big a perf impact (constructing the std::wstring object? 

---
In reply to: [92735069](https://github.com/Microsoft/WinObjC/pull/1558#discussion_r92735069) [](ancestors = 92735069)